### ISSUE TITLE
Issues with Zhili Clique

### ISSUE THREAD

**Issue by KirovIsDead** [jespertjee/FR#115]
**Explanation of the issue:**
After playing Zhili Clique I've found several issues with this country:
1) During and after war with Japan national spirit "Japanese backing" still exists.
2) Seems that Japan can't progress down their focus tree if they lost war with Zhili.
3) Sun Chuanfang is still a fieldmarchal after taking democratic(Cao Kun) path.
4) Focus "Topple the Southern pretenders" will be bypassed if EVEN ONE of southern warlords out of five doesn't exist, and player won't get CBs. Instead focus should be bypassed only when NONE of them exist.
5) China has a core on Mongolian Northern Chahar; if Beiyang goverment is proclaimed and Mongolia collapses under Soviet invasion, there should be an option to take that state without war with Soviets.
6) After full playthrough there was no way to rid the nation of illiteracy, the focus "Educational standards in the army" had no effect.
7) When French cave to demands, East-Asien doesn't cede land.
8) After wars of reconquest Zhili doesn't get cores on Tibet, Xinjiang and Mongolia
9) If Cao Kun's path taken, there's now way to get rid of national spirit "A clique of traitors".
10) When Cao Kun rises to power authoritarian democrats should get a boost in ideology.
11) Spirit "Grace of the Tiger" giving debuffs even after completing military branch of the focus tree.
12) If you proclaim Beiyang goverment through focus tree, you still can take a decision to proclaim it again in decision tab, and country name will revert back to Zhili Clique.
13) Invitations to the faction you've created don't work properly.

**Quick questions:**

OS:
HoI4 version: 1.5.4
Führerreich version: 0.2
Using Steam? Yes.
Were you in multiplayer? No.
List other mods used (including FR Music mod): None.
Have you changed any files, outside of mods used? No.
Checksum (four-letter code at the bottom of your title menu): 63a8
Which DLC expansions do you NOT have? I have all expansions,

**Steps to reproduce:**

1. Play Zhili Clique

2. a) Win the war with Japan
b) Choose Cao Kun path

**Other info:**

**Possible cause:**

**Log file:**

**Screenshots / Video:**

**Save game:**

**Comment by KhanThymos** [comment]
Thank you for your great issue, this has been a huge help in bug fixing! Some of the issues you listed are actually intended, most prominently the cores on Mongolia, Xinjiang, and Tibet as those are not Chinese.

**Comment by KirovIsDead** [comment]
Yes, these regions aren't Chinese in ethnic sense, but they were all part of Qing, so it's safe to assume that any able post-Qing goverment would want them back under central goverment direct control. Of course, population of these states would not be too happy immediatly after becoming part of China, and i feel like nationalist and religious dissent of minorities and the process of combating it and actually coring the land should be represented by national spirits and decisions/focuses. And, if land can't be cored, maybe there should be an option to install some kind of military puppet govement, like in Korea.
Anyway, glad I could help with bug fixing!
